blocks/columns/columns.css

```diff
@@ -224,6 +275,10 @@
     margin: 0;
 }
 
+.section:not(.leadspace) .image-end picture, .section:not(.leadspace) .image-start picture {
+    display: none;
+    }
```

Comment:
This looks similar to an existing style.

```
.news .columns.quarter-circle picture {
    display: none;
}
```

Is it possible to force ALL `.columns.quarter-circle picture` to not display on mobile?

Reply:
OK, deleted older (redundant) code. But no to force all to not display per the Figma designs. That's why I have the "not leadspace".